src/app/backend/apihandler.go

```diff
@@ -18,17 +18,35 @@ import (
 	"log"
 	"net/http"
 	"strconv"
+	"strings"
 
 	restful "github.com/emicklei/go-restful"
 	client "k8s.io/kubernetes/pkg/client/unversioned"
 )
 
+func WsLogger() restful.FilterFunction {
```

Comment:
If this is a lot of work, let's submit as it is. And later add response logging, when we find stronger need.